### ISSUE TITLE
extraneous: color-convert@1.0.0

### ISSUE THREAD

**Issue by akoskm** [nodesecurity/nsp#92]
I'm not sure if this is an issue with nsp but it happens from version > 2.2.0.

`npm shrinkwrap` fails with:

```
npm ERR! Linux 3.13.0-83-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "shrinkwrap"
npm ERR! node v4.4.1
npm ERR! npm  v2.14.20

npm ERR! Problems were encountered
npm ERR! Please correct and try again.
npm ERR! extraneous: color-convert@1.0.0 /home/akoskm/project/node_modules/nsp/node_modules/chalk/node_modules/ansi-styles/node_modules/color-convert
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/akoskm/project/npm-debug.log
```

Reverting to 2.2.0 makes this error disappear.

**Comment by nlf** [comment]
this should be resolved with 2.3.0 (published this morning), feel free to reopen if you're still having problems.

**Comment by akoskm** [comment]
Thanks, `npm shrinkwrap` work fine with 2.3.0.
